compiler/noirc_frontend/src/hir/comptime/value.rs

```diff
@@ -206,6 +208,7 @@ impl Value {
             | Value::TraitConstraint(_)
             | Value::TraitDefinition(_)
             | Value::FunctionDefinition(_)
+            | Value::Zeroed(_)
```

Comment:
Could this be supported by returning `std::unsafe::zeroed::<T>()`?

Reply:
Yes, it's just that it isn't easy to access. I'd like to leave it to a later PR to automatically find zeroed in the stdlib and save it in the interner.